builtin/providers/vsphere/resource_vsphere_virtual_machine.go

```diff
@@ -1,6 +1,7 @@
 package vsphere
 
 import (
+	"errors"
```

Comment:
You are correct. I will change this over the weekend (it's close to midnight here, getting quite sleepy. :smile: )